### PR TITLE
Loky 2.8.0

### PR DESCRIPTION
**Pull request by ogrisel**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ogrisel** [comment]
@conda-forge-admin, please rerender

**Comment by ogrisel** [comment]
@conda-forge-admin, please rerender

**Comment by ogrisel** [comment]
Ok this looks good now. I messed up with creating a `loky-2.8.0` branch in the repo first before issuing a PR from my fork to that branch...

I deleted the branch in the main repo, pointed this PR back to master and rerendered.

**Comment by ogrisel** [comment]
It this the first release that drops Python 2 support on loky in conda-forge? We should review that point carefully before merging to avoid the problem we had with cloudpickle.

**EDIT** I am stupid, we have not merged https://github.com/joblib/loky/pull/237 yet.

I was confused by https://github.com/conda-forge/loky-feedstock/pull/17/files#diff-b41481fa6aa06c79dccd3b6158244e26 which is probably unrelated.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 About loky
 ==========
 
-Home: https://github.com/joblib/loky
+Home: https://github.com/tomMoral/loky
```
Comment (tomMoral) [inline README.md]: There seems to be an issue here as the README is updated in the wrong direciton

Comment (ogrisel) [reply README.md]: hum... probably a metadata file is wrong somewhere.

**Comment by tomMoral** [comment]
I though we were dropping python2 support in `loky3.0`?

**Comment by ogrisel** [comment]
> I though we were dropping python2 support in `loky3.0`?

Yes, it's just me that was confused by task switching between projects. I edited: https://github.com/conda-forge/loky-feedstock/pull/17#issuecomment-628706571

**Comment by ogrisel** [comment]
@conda-forge-admin, please rerender

Thank you for dealing with me tireless little bot.

**Comment by tomMoral** [comment]
There is a concurrent auto-PR: https://github.com/conda-forge/loky-feedstock/pull/18

It seems to be the same, with the typo in the link in the readme so in favor of merging this one.

**Comment by ogrisel** [comment]
Merged!

**Pull request merged**